postgres/client.sls

```diff
@@ -23,11 +23,14 @@ postgresql-client-libs:
     - refresh: True
     - require:
       - pkgrepo: postgresql-repo
+       {% if grains.os_family == 'Suse' %}
+      - cmd: postgresql-repo
+       {% endif %}
 {%- endif %}
 
 {%- if 'bin_dir' in postgres %}
 
-# Make client binaries available in $PATH
+  # Make client binaries available in $PATH
```

Comment:
No need for indenting these comments, since they're part of rendered YAML, not Jinja.
It would look better in debug log when they are aligned with states they describe.